test/e2e/storage/testsuites/provisioning.go

```diff
@@ -712,6 +712,32 @@ func StopPod(c clientset.Interface, pod *v1.Pod) {
 	e2epod.DeletePodWithWait(c, pod)
 }
 
+// StopPodAndDependents first tries to log the output of the pod's container,
+// then deletes the pod and waits for that to succeed. Also waits for all owned
+// resources to be deleted.
+func StopPodAndDependents(c clientset.Interface, pod *v1.Pod) {
```

Comment:
A better place for this may be https://github.com/kubernetes/kubernetes/tree/f5c7ce96ce2644fc229e6d5479a3ca976ecb1b83/test/e2e/storage/utils. (In general, these files could use some reorganization)

Reply:
@msau42 agreed. I plan on circling back for cleanup but would like to get these tests working again ASAP. I'll open a tracking issue and assign myself 🙂

Reply:
Opened issue #93347